responses/user_response.go

```diff
@@ -16,6 +12,7 @@ type ProfileResponses struct {
 }
 
 type GetProfileResponses struct {
-	Status      int    `json:"status"`
-	ProfileInfo bson.M `bson:"message,omitempty"`
+	Status  int                 `json:"status"`
+	Message string              `json:"message,omitempty"`
+	User    map[string]interface{} `json:"user,omitempty"`
```

Comment:
Creating a proper struct to ensure we have the only fields we need is a better idea. If you can please do that. Otherwise, this works for now.